packages/react/src/core/ContextMenu.tsx

```diff
@@ -0,0 +1,132 @@
+import { useRef, useState, type ComponentProps } from 'react';
+import { Root, Portal, Content, Item, Trigger } from '@radix-ui/react-context-menu';
+import type { I18n } from '@tutorialkit/types';
+
+interface FileChangeEvent {
+  type: 'FILE' | 'FOLDER';
+  method: 'ADD' | 'REMOVE' | 'RENAME';
```

Comment:
Added the `REMOVE` and `RENAME` here even though they are not yet implemented. This kind of API allows us to extend the functionality later without causing breaking changes.